athena-jdbc/src/main/java/com/amazonaws/connectors/athena/jdbc/saphana/SAPHANAQueryStringBuilder.java

```diff
@@ -0,0 +1,69 @@
+/*-
+ * #%L
+ * athena-jdbc
+ * %%
+ * Copyright (C) 2019 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.jdbc.saphana;
+
+import com.amazonaws.athena.connector.lambda.domain.Split;
+import com.amazonaws.connectors.athena.jdbc.manager.JdbcSplitQueryBuilder;
+import com.google.common.base.Strings;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Extends {@link JdbcSplitQueryBuilder} and implements SAPHANA specific SQL clauses for split.
+ *
+ * SAPHANA provides named partitions which can be used in a FROM clause.
+ */
+public class SAPHANAQueryStringBuilder
```

Comment:
naming convention

Reply:
Same as above

Reply:
Thats fine, choose a name that uses CammelCase

Reply:
Fixed, please check